scheduler/roles/slurm/tasks/validation.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Slurm control and node group to contain at least 1 node
+  ansible.builtin.assert:
+    that:
+      - "'slurm_control_node' in groups"
```

Comment:
add slurm validations in cluster_validation role

Reply:
That is not changed, its still there

Reply:
validation  need to move out of slurm role as other validation are present in the same role. all validation should be invoked first during omnia.yml execution.